### PR TITLE
*: update doc for generated columns GA

### PR DESCRIPTION
**Pull request by ran-huang**
### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:https://github.com/pingcap/docs-cn/pull/13383
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-create-table.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ A [primary key](/constraints.md#primary-key) is a column or a set of columns in
 
 > **Note:**
 >
-> The default definition of **primary key** in TiDB is different from that in [InnoDB](https://mariadb.com/kb/en/innodb/)(the common storage engine of MySQL). 
+> The default definition of **primary key** in TiDB is different from that in [InnoDB](https://mariadb.com/kb/en/innodb/)(the common storage engine of MySQL).
```
Comment (dveeden) [inline develop/dev-guide-create-table.md:105]:
```suggestion
> The default definition of **primary key** in TiDB is different from that in [InnoDB](https://dev.mysql.com/doc/refman/8.0/en/innodb-storage-engine.html)(the common storage engine of MySQL).
```

Comment (dveeden) [reply develop/dev-guide-create-table.md:105]: Looks like this same sentence and links are also in `develop/dev-guide-schema-design-overview.md`

Comment (ran-huang) [reply develop/dev-guide-create-table.md]: Thank you! Just fixed it.

File: develop/dev-guide-create-table.md [hunk 2]
```diff
@@ -102,7 +102,7 @@ A [primary key](/constraints.md#primary-key) is a column or a set of columns in
 
 > **Note:**
 >
-> The default definition of **primary key** in TiDB is different from that in [InnoDB](https://mariadb.com/kb/en/innodb/)(the common storage engine of MySQL). 
+> The default definition of **primary key** in TiDB is different from that in [InnoDB](https://dev.mysql.com/doc/refman/8.0/en/innodb-storage-engine.html)(the common storage engine of MySQL).
```
Comment (qiancai) [inline develop/dev-guide-create-table.md:105]: https://github.com/pingcap/docs-cn/pull/13383/files 中文 PR 是否需要对应修改下

File: develop/dev-guide-schema-design-overview.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ There are two common types of indexes:
 
 > **Note:**
 >
-> In TiDB, the default definition of **Primary Key** is different from that in [InnoDB](https://mariadb.com/kb/en/innodb/) (a common storage engine of MySQL).
+> In TiDB, the default definition of **Primary Key** is different from that in [InnoDB](https://dev.mysql.com/doc/refman/8.0/en/innodb-storage-engine.html) (a common storage engine of MySQL).
```
Comment (qiancai) [inline develop/dev-guide-schema-design-overview.md:43]: https://github.com/pingcap/docs-cn/pull/13383/files 中文 PR 是否需要对应修改下

**Review by qiancai — APPROVED** [review]
other LGTM

File: develop/dev-guide-schema-design-overview.md (hunk 3 above)
Comment (ran-huang) [reply develop/dev-guide-schema-design-overview.md:43]: done. Thanks!

**Pull request merged**